modules/igv/1.0/config/default.yaml

```diff
@@ -0,0 +1,118 @@
+
+            generate_batch_script:
+                padding: 100 # Base pairs upstream and downstream of variant position
+                max_height: 1000 # Maximum height of snapshot
+                sleep_timer: 2000 # Batch scripts with more options may require longer sleep intervals
+                igv_options:
+                    # Presets for IGV snapshots
+                    # Available igv options: https://github.com/igvteam/igv/wiki/Batch-commands
+                    default: ["preference SAM.COLOR_BY READ_STRAND", "preference SAM.SHOW_CENTER_LINE TRUE", "preference SAM.SHADE_BASE_QUALITY true", "preference SAM.DOWNSAMPLE_READS FALSE", "preference SAM.ALLELE_THRESHOLD 0.05", "sort"]
+                    pairs: ["viewaspairs", "preference SAM.COLOR_BY READ_STRAND", "preference SAM.SHOW_CENTER_LINE TRUE", "preference SAM.SHADE_BASE_QUALITY true", "preference SAM.DOWNSAMPLE_READS FALSE", "preference SAM.ALLELE_THRESHOLD 0.05", "sort QUALITY"]
+
+            igv_presets: ["default"] # Available options: "default" "pairs"
+
+            xvfb_parameters:
+                # Server options for running xvfb
+                server_number: "99"
+                server_args: ""
+
+            quality_control:
+                # Truncated heights that have been previously observed for dimensions 1920x1080x24 
+                truncated: [506,533,545,547,559,570]
+                # Kurtosis and skewness values observed in blank snapshots at different height values
+                blank:
+                    "547":
+                        kurtosis: 18.5
+                        skewness: -4
+                    "559":
+                        kurtosis: 18.2
+                        skewness: -4
+                # Previously observed heights of snapshots that fail IGV
+                failed: [506,533]
+
+        scripts:
+            format_regions: "etc/format_regions.py"
+            filter_script: "etc/filter_maf.py"
+            region_liftover_script: "{MODSDIR}/etc/liftover_regions.sh"
+            batch_script_per_variant: "etc/generate_batch_script_per_variant.py"
+            quality_control: "etc/quality_control.py"
+
+        scratch_subdirectories: []
+
+        conda_envs:
+            crossmap: "{MODSDIR}/envs/crossmap.yaml"
+            wget: "{REPODIR}/envs/wget/wget-1.20.1.yaml"
+
+        threads: 4
+
+        resources:
+            _igv_liftover_regions:
+                mem_mb: 2000
+            _igv_run:
+                mem_mb: 2500
+            _igv_quality_control:
+                mem_mb: 2500
+            
+        pairing_config:
+            genome:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            capture:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+
+    slms_3:
```

Comment:
can you instead just include config from slms-3?
I think this will conflict if you have snakefile that will attempt to run both slms-3 and this module because snakemake won't like duplicated config values - or will just use whatever is imported last so may have unexpected consequence in that scenario

Reply:
Yes good idea, I removed this part from the config